### PR TITLE
On new OmniNav, apply minor post-release changes.

### PR DESCRIPTION
**Pull request by kaylaziegler**
Story: https://trello.com/c/bUBsYnoq

This AC has not been addressed yet:
> On mobile devices, the search box does a weird thing where the border around the input box breaks a little. Ideally, it would be a perfect rectangle instead of a broken one.

Thinking of moving it to another story so it doesn't hold up the other 9 fixes in this story.

### CODE REVIEW

File: app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss [hunk 1]
```diff
@@ -10,32 +10,18 @@ html.omni-nav-v2 {
     padding-top: 120px;
   }
 
-  @include media($large-screen) {
+  @include media($xl-screen) {
     padding-top: 60px;
   }
 
-  /* branded omninav always has primary nav stacked on top of global nav */
```
Comment (kaylaziegler) [inline app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss]: Unbranded and branded pages used to have different breakpoints but now they share one, which is why there are quite a few deletions in this area

File: app/controllers/omninav_controller.rb [hunk 2]
```diff
@@ -5,33 +5,48 @@ def demo
...
-      # [ directory (assume leading /), menu label, class/image id, path, brand text ]
-      [ 'business', 'ARGYROS SCHOOL', 'asbe', 'business/index.aspx', 'Argyros School of Business' ],
-      [ 'education', 'ATTALLAH COLLEGE', 'education', 'education/index.aspx' ],
-      [ 'dodge', 'DODGE COLLEGE', 'dodge', 'dodge/index.aspx' ],
-      [ 'crean', 'CREAN COLLEGE', 'crean', 'crean/index.aspx' ],
-      [ 'wilkinson', 'WILKINSON COLLEGE', 'wilkinson', 'wilkinson/index.aspx' ],
-      [ 'copa', 'CoPA', 'copa', 'copa/index.aspx' ],
-      [ 'pharmacy', 'CUSP', 'cusp', 'pharmacy/index.aspx' ],
-      [ 'law', 'FOWLER SCHOOL OF LAW', 'fowler', 'law/index.aspx', 'Fowler School of Law' ]
+      # [ directory (assume leading /), menu label, class/image id ]
```
Comment (kaylaziegler) [inline app/controllers/omninav_controller.rb]: Basically just copied the umbrella category map from production. I reworked this controller so we could more easily test the new OmniNav with different templates.

File: app/views/omni_nav_v2/_utility_nav.html.erb [hunk 3]
```diff
@@ -39,23 +39,23 @@
...
                   </a></li>
-                  <li><a href="http://pinterest.com/chapmanu/" target="_blank">
-                    <svg class="icon icon-pinterest" xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32"><title>  pinterest2</title><path d="M16 0c-8.8 0-16 7.2-16 16s7.2 16 16 16 16-7.2 16-16-7.2-16-16-16zM16 29.9c-1.4 0-2.8-0.2-4.1-0.6 0.6-0.9 1.4-2.4 1.7-3.6 0.2-0.6 0.9-3.3 0.9-3.3 0.5 0.9 1.8 1.6 3.2 1.6 4.2 0 7.2-3.8 7.2-8.6 0-4.6-3.7-8-8.5-8-6 0-9.2 4-9.2 8.4 0 2 1.1 4.6 2.8 5.4 0.3 0.1 0.4 0.1 0.5-0.2 0-0.2 0.3-1.1 0.4-1.6 0-0.1 0-0.3-0.1-0.4-0.6-0.7-1-2-1-3.2 0-3.1 2.3-6 6.3-6 3.4 0 5.8 2.3 5.8 5.6 0 3.8-1.9 6.4-4.4 6.4-1.4 0-2.4-1.1-2-2.5 0.4-1.6 1.2-3.4 1.2-4.6 0-1.1-0.6-1.9-1.8-1.9-1.4 0-2.5 1.4-2.5 3.4 0 1.2 0.4 2.1 0.4 2.1s-1.4 5.8-1.6 6.9c-0.3 1.2-0.2 2.9 0 4-5.2-2-8.8-7-8.8-12.9 0-7.7 6.2-13.9 13.9-13.9s13.9 6.2 13.9 13.9c0 7.7-6.2 13.9-13.9 13.9z"/></svg>
+                  <li><a href="http://www.linkedin.com/company/10701?trk=tyah" target="_blank">
```
Comment (kaylaziegler) [inline app/views/omni_nav_v2/_utility_nav.html.erb]: Social icons didn't match production, and one of the AC involved them so I made cascade assets match.

File: app/assets/javascripts/omni-nav-v2.js [hunk 4]
```diff
@@ -446,11 +445,33 @@ var OmniNav2 = (function() {
...
+          $("table.gstl_50.gssb_c").find(".gsc-completion-container").on("click", onAutocompleteClick);
+          utilityAutocompleteBound = true;
+        }
+        else if(!primaryAutocompleteBound) {
+          $("table.gstl_51.gssb_c").find(".gsc-completion-container").on("click", onAutocompleteClick);
+          primaryAutocompleteBound = true;
+        }
+      }
+
+      var onAutocompleteClick = function(e) {
```
Comment (kaylaziegler) [inline app/assets/javascripts/omni-nav-v2.js]: Added a story to address a minor lingering issue with autocomplete: https://trello.com/c/rQcWMSLg

File: app/assets/javascripts/omni-nav-v2.js [hunk 5]
```diff
@@ -446,11 +445,33 @@ var OmniNav2 = (function() {
...
+        setTimeout(showSearchResults, 100);
```
Comment (tatwell) [inline app/assets/javascripts/omni-nav-v2.js]: Why the timeout here?

Comment (kaylaziegler) [reply app/assets/javascripts/omni-nav-v2.js]: Oops guess I accidentally deleted that code comment, I'll re-add it. It has to pause for the autocomplete term to be added to the DOM first

**Pull request merged**